presentation-compiler/test/dotty/tools/pc/tests/completion/CompletionRelease8Suite.scala

```diff
@@ -0,0 +1,31 @@
+package dotty.tools.pc.tests.completion
+
+import dotty.tools.pc.base.BaseCompletionSuite
+
+import org.junit.Test
+import org.junit.Before
+import java.nio.file.Path
+import dotty.tools.pc.utils.JRE
+
+class CompletionRelease8Suite extends BaseCompletionSuite:
+
+  override protected def scalacOptions(classpath: Seq[Path]): Seq[String] =
+    "-release:8" +: super.scalacOptions(classpath)
+
+  @Before
+  def beforeMethod(): Unit =
+    org.junit.Assume.assumeTrue(JRE.getJavaMajorVersion >= 8)
+
+  @Test def noJvm11Symbols =
+    check(
+      """
+        |object A {
+        |  "".repea@@
```

Comment:
How is this actually working? I think I actually had to turn it off and -release options is removed:

https://github.com/scala/scala3/blob/5e836060fb16e08b4961ec7c70484786633aa164/presentation-compiler/src/main/dotty/tools/pc/ScalaPresentationCompiler.scala#L61

Reply:
Sorry, but I don't understand the question. Are you asking about how the test is implemented or how are we handling the release flag ?

Reply:
How are the tests working? And maybe we need to stop removing the flag?

Reply:
Do you remember why we added this check in the first place ?

Reply:
It might have been interfering with some other options and it seemed not important for frontend phases https://github.com/scalameta/metals/pull/3812

Reply:
I just tested it and seems to work ok with that removed.